test/boost/sstable_3_x_test.cc

```diff
@@ -3035,8 +3035,7 @@ static flat_mutation_reader_v2 compacted_sstable_reader(test_env& env, schema_pt
 
     auto desc = sstables::compaction_descriptor(std::move(sstables), default_priority_class());
     desc.creator = [&] (shard_id dummy) {
-        compacted_sst = env.make_sstable(s, tmp.path().string(), new_generation,
-                         sstables::sstable_version_types::mc, sstable::format_types::big, 4096);
```

Comment:
This defaults the buffer-size of 4096. Dunno if this is was test's intention to use smaller buffer